docs/multiplayer.md

```diff
@@ -209,17 +209,22 @@ Last updated:    2011-02-16
 
 
 ## 6.0) Troubleshooting
-
- - My advertising server does not show up in list at servers.openttd.org
-     Run openttd with the '-d net=2' parameter. That will show which incoming
-     communication is received, whether the replies from the master server or
-     communication from an admin tool reach the programme. See section 1
-     'Starting a server' further up for the ports and protocols used by OpenTTD.
-     The ports can be configured in the config file.
-
- - My advertising server warns a lot about getaddrinfo taking N seconds
-     This could be a transient issue with your (local) DNS server, but if the
-     problem persists there is likely a configuration issue in DNS resolving
-     on your computer. This seems to be a common configuration issue for
-     Docker instances, where the DNS resolving waits for a time out of usually
-     5 seconds.
+### My advertising server does not show up in list at servers.openttd.org
+Run openttd with the `-d net=2` parameter.
+That will show which incoming communication is received, whether the replies
+from the master server or communication from an admin tool reach the programme.
+See [1.0) Starting a server](#10-starting-a-server) for the ports and protocols
+used by OpenTTD.
+The ports can be configured in the config file.
+
+### My advertising server warns a lot about getaddrinfo taking N seconds
+This could be a transient issue with your (local) DNS server, but if the
+problem persists there is likely a configuration issue in DNS resolving on your
+computer.
+#### Docker containers
+This is an issue for dual-stack Docker containers, in which there is no default
+IPv6 resolver, when IPv6 traffic is preferred (attempted before) over IPv4,
+where it can be observed name resolution times out after a delay (5 seconds
+default on GNU/Linux).
+The solution is then to force custom name servers for said containers to be a
+list of IPv4 ones.
```

Comment:
Saying "do this thing" without actually saying how to do it (and a cursory google doesn't reveal anything obvious), isn't particularly useful. Can you expand on this?